examples/contracts/llm_erc20.py

```diff
@@ -34,24 +35,20 @@ def transfer(self, amount: int, to_address: str) -> None:
 "transaction_error": str,             // Empty if transaction is successful
 "updated_balances": object<str, int>  // Updated balances after the transaction
 }}
+
 It is mandatory that you respond only using the JSON format above,
 nothing else. Don't include any other words or characters,
 your output must be only JSON without any formatting prefix or suffix.
-This result should be perfectly parsable by a JSON parser without errors."""
-        print(prompt)
-
-        def compute_updated_balances():
-            result = gl.exec_prompt(prompt)
-            result = result.replace("```json", "").replace("```", "")
-            return result
+This result should be perfectly parsable by a JSON parser without errors.
+"""
 
-        final_result = gl.eq_principle_prompt_noncomparative(
-            compute_updated_balances,
-            """
-The new_balance of the sender should have decreased
-in the amount sent and the new_balance of the receiver should have
-increased by the amount sent. Also, the total sum of all balances
-should remain the same before and after the transaction""",
+        final_result = (
```

Comment:
@kp2pml30 could you please explain the non-comparative eq principle a bit more? it would also be great to add them to the docs :)

Reply:
I am trying to get some feedback from Edgars, as soon as he answers I will add

Reply:
@kp2pml30, is this ready, or did you get the feedback that you needed?